lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -783,12 +787,12 @@ namespace Js
         FunctionCodeGenRuntimeData **data = this->GetCodeGenRuntimeData();
         if (data != nullptr)
         {
-            this->UpdateActiveFunctionsForOneDataSet(pActiveFuncs, data, this->GetProfiledCallSiteCount());
+            this->UpdateActiveFunctionsForOneDataSet(pActiveFuncs, nullptr, data, this->GetProfiledCallSiteCount());
```

Comment:
>nullptr [](start = 67, length = 7)

I don't have much insights in this code, but won't this always go inside the if-condition you have above `inlineeData != parentData`?